jax/_src/checkify.py

```diff
@@ -589,13 +590,14 @@ def isnan(x):
 
 def gather_error_check(error, enabled_errors, operand, start_indices, *,
                        dimension_numbers, slice_sizes, unique_indices,
-                       indices_are_sorted, mode, fill_value):
+                       indices_are_sorted, mode, fill_value, check):
   out = lax.gather_p.bind(
       operand, start_indices, dimension_numbers=dimension_numbers,
       slice_sizes=slice_sizes, unique_indices=unique_indices,
-      indices_are_sorted=indices_are_sorted, mode=mode, fill_value=fill_value)
+      indices_are_sorted=indices_are_sorted, mode=mode, fill_value=fill_value,
+      check=check)
```

Comment:
Should this be `False` since we're running the check?

Reply:
ah yeah good catch!